packages/flutter/lib/src/foundation/binding.dart

```diff
@@ -170,35 +170,48 @@ abstract class BindingBase {
   ///  * [DebugReassembleConfig], which describes the configuration.
   static DebugReassembleConfig? debugReassembleConfig;
 
-  /// The main window to which this binding is bound.
+  /// Deprecated. Will be removed in a future version of Flutter.
   ///
-  /// A number of additional bindings are defined as extensions of
-  /// [BindingBase], e.g., [ServicesBinding], [RendererBinding], and
-  /// [WidgetsBinding]. Each of these bindings define behaviors that interact
-  /// with a [ui.SingletonFlutterWindow].
+  /// This property has been deprecated to prepare for Flutter's upcoming
+  /// support for multiple views and multiple windows.
   ///
-  /// Each of these other bindings could individually access a
-  /// [ui.SingletonFlutterWindow] statically, but that would preclude the
-  /// ability to test its behaviors with a fake window for verification
-  /// purposes. Therefore, [BindingBase] exposes this
-  /// [ui.SingletonFlutterWindow] for use by other bindings. A subclass of
-  /// [BindingBase], such as [TestWidgetsFlutterBinding], can override this
-  /// accessor to return a different [ui.SingletonFlutterWindow] implementation,
-  /// such as a [TestWindow].
-  ///
-  /// The [window] is a singleton meant for use by applications that only have a
-  /// single main window. In addition to the properties of [ui.FlutterView],
-  /// [window] provides access to platform-specific properties and callbacks
-  /// available on the [platformDispatcher].
-  ///
-  /// For applications designed for more than one main window, prefer using the
-  /// [platformDispatcher] to access available views via
-  /// [ui.PlatformDispatcher.views].
-  ///
-  /// However, multiple window support is not yet implemented, so currently this
-  /// provides access to the one and only window.
-  // TODO(gspencergoog): remove the preceding note once multi-window support is
-  // active.
+  /// It represents the main view for applications where there is only one
+  /// view, such as applications designed for single-display mobile devices.
+  /// If the embedder supports multiple views, it points to the first view
+  /// created which is assumed to be the main view. It throws if no view has
+  /// been created yet or if the first view has been removed again.
+  ///
+  /// The following options exists to migrate code that relies on accessing
+  /// this deprecated property:
+  ///
+  /// If a [BuildContext] is available, consider looking up the current
+  /// [FlutterView] associated with that context via [View.of]. It gives access
```

Comment:
Consider:

```suggestion
  /// If a [BuildContext] is available, you can get the context's
  /// current [FlutterView] via [View.of]. It gives access
```

I don't feel strongly about this

Reply:
I am gonna leave this as-is since our style guide doesn't like to use the word "you": https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-the-passive-voice-recommend-do-not-require-never-say-things-are-simple